test/specs/collections/Form/FormFieldCheckbox-test.js

```diff
@@ -5,7 +5,9 @@ import Checkbox from 'src/modules/Checkbox/Checkbox'
 import * as common from 'test/specs/commonTests'
 
 describe('FormCheckbox', () => {
-  common.isConformant(FormCheckbox)
+  common.isConformant(FormCheckbox, {
+    ignoredProps: ['type'],
```

Comment:
Nit: given that the conformance test covers many things, can we rename this option to specify that it only ignores the prop in typings?

Perhaps `ignoreTypingsProps` or similar?